src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -622,9 +631,9 @@ type Get(nameChain: string) =
     | OneWayToSource _ -> GetError.OneWayToSource |> Error
     | OneWaySeq { Values = vals } -> vals.GetCollection () |> Ok
     | Cmd cmd -> cmd |> box |> Ok
-    | SubModel { Vm = vm } -> vm.Value |> ValueOption.toObj |> box |> Ok
-    | SubModelWin { VmWinState = vm } ->
-        vm.Value
```

Comment:
Nit: keep on one line

Reply:
Pushed an update

Reply:
I don't recall what was here when I made my first comment. That is a downside of force pushing (though I still prefer force pushing overall).

I want this diff to be smaller.  I want the code that replaces this line to remain on its own line.

Hopefully this comment isn't the opposite of what I meant in my first comment.

Reply:
@TysonMN [Here is the diff that I pushed as a result of this comment](https://github.com/elmish/Elmish.WPF/compare/309c99c78172089debd9c92c495b27c81241b36e..72c0684b59b040f468788a0d45144a3b90043ded)

Reply:
I am not sure what happened.  I thought I commented about change from something like

```fsharp
a
|> b
|> c
```

to

```fsharp
a' |> b |> c
```

but I can't find any evidence of that.

Oh, well.  I pushed the changed I wanted here (which appears to be what you originally had).  Sorry about that.